MANUAL.md

```diff
@@ -0,0 +1,192 @@
+# Status Swarm Manual
+
+This document attempts to outline a broad set of guidelines so we can operate swarms effectively as an organization.
+
+What we are working towards is an entirely decentralized organization, where ideas and swarms are funded by any user, stakeholder, in the system. This is a non-trivial problem in terms of incentive design. To get there we are currently using a two-pronged approach. This document focuses purely on the first point:
+
+1.  Formalize swarm lead process and come to a shared agreement about role expectations
+    
+2.  Run incentive/reputation design experiments through Open Bounty
+
+Additionally, there will likely be some changes to ideas repository to
+facilitate the running of a swarm.
+
+### Swarm lifecycle
+
+A swarm can be in the following states: draft, in progress, completed or aborted:
+
+-   A swarm starts off as an idea in draft mode
```

Comment:
If people find tools like Google Docs useful then go for it, but it should probably be captured as a PR with draft state (or in progress state if it is ready to go). Don't think we need to overly specify this.

Reply:
To me issue could be something like a problem people have (with idea as stated, or maybe something that isn't idea yet but should be), and then someone could file a PR against this. This would be consistent with how it works in code generally.